ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -875,3 +875,69 @@ def test_tensorflow_gather_nd(
         indices=indices,
         batch_dims=batch_dims,
     )
+
+
+@st.composite
+def _pad_helper(draw):
+    mode = draw(
+        st.sampled_from(
+            [
+                "constant",
+                "reflect",
+                "symmetric",
+            ]
+        )
+    )
+    dtype, input, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
+            ret_shape=True,
+            min_num_dims=1,
+            min_value=-100,
+            max_value=100,
+        ).filter(lambda x: x[0][0] not in ["float16", "bfloat16"])
+    )
```

Comment:
We shouldn't be filtering data types, the `helpers.get_dtypes` won't generate invalid dtypes if the function was marked that it doesn't support data type `x`, instead of filtering the data types, you should flag that the function doesn't support the specific data type.